cli/bpmetadata/tfconfig.go

```diff
@@ -279,10 +310,30 @@ func getBlueprintRequirements(rolesConfigPath, servicesConfigPath string) (*Blue
 		return nil, err
 	}
 
-	return &BlueprintRequirements{
-		Roles:    r,
-		Services: s,
-	}, nil
+	if versionsConfigPath != "" {
```

Comment:
The file path I belive is static and will always be there but we should do something like https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/3081bdc1f6710674330c337aac74484df3e4daa1/cli/util/file.go#L97 to check if its exists.

Reply:
I applied https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/3081bdc1f6710674330c337aac74484df3e4daa1/cli/bpmetadata/path.go#L23 here, WDYT?